### ISSUE TITLE
[BUG]: Multipart-form data not being accepted

### ISSUE THREAD

**Issue by mr-staun** [serengil/deepface#1403]
### Before You Report a Bug, Please Confirm You Have Done The Following...

- [X] I have updated to the latest version of the packages.
- [X] I have searched for both [existing issues](https://github.com/serengil/deepface/issues) and [closed issues](https://github.com/serengil/deepface/issues?q=is%3Aissue+is%3Aclosed) and found none that matched my issue.

### DeepFace's version

0.0.94

### Python version

3.8.12

### Operating System

Fedora 41

### Dependencies

absl-py==2.1.0
astunparse==1.6.3
beautifulsoup4==4.12.3
blinker==1.8.2
cachetools==5.5.0
certifi==2024.12.14
charset-normalizer==3.4.0
click==8.1.7
# Editable install with no version control (deepface==0.0.94)
-e /app
filelock==3.16.1
fire==0.7.0
Flask==3.0.3
Flask-Cors==5.0.0
flatbuffers==24.3.25
gast==0.4.0
gdown==5.2.0
google-auth==2.37.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.68.1
gunicorn==23.0.0
h5py==3.11.0
idna==3.10
importlib_metadata==8.5.0
itsdangerous==2.2.0
Jinja2==3.1.4
keras==2.13.1
libclang==18.1.1
Markdown==3.7
MarkupSafe==2.1.5
mtcnn==0.1.1
numpy==1.22.3
oauthlib==3.2.2
opencv-python==4.9.0.80
opt_einsum==3.4.0
packaging==24.2
pandas==2.0.3
Pillow==9.0.0
protobuf==4.25.5
pyasn1==0.6.1
pyasn1_modules==0.4.1
PySocks==1.7.1
python-dateutil==2.9.0.post0
pytz==2024.2
requests==2.32.3
requests-oauthlib==2.0.0
retina-face==0.0.17
rsa==4.9
six==1.17.0
soupsieve==2.6
tensorboard==2.13.0
tensorboard-data-server==0.7.2
tensorflow==2.13.1
tensorflow-estimator==2.13.0
tensorflow-io-gcs-filesystem==0.34.0
termcolor==2.4.0
tqdm==4.67.1
typing_extensions==4.5.0
tzdata==2024.2
urllib3==2.2.3
Werkzeug==3.0.6
wrapt==1.17.0
zipp==3.20.2

### Reproducible example

```Python
Run the docker container, then attempt to execute this curl command outside the container: `curl -X POST http://0.0.0.0:5005/represent -H "Content-Type: multipart/form-data" -F "model_name=ArcFace" -F "im
g=@Cruelty_of_life.jpg"`
```

### Relevant Log Output

```html
<!doctype html>
<html lang=en>
<title>415 Unsupported Media Type</title>
<h1>Unsupported Media Type</h1>
<p>Did not attempt to load JSON data because the request Content-Type was not &#39;application/json&#39;.</p>
```

### Expected Result

I expected to obtain the embeddings from the given image

### What happened instead?

The application expected a JSON format

### Additional Info

This was requested in #1382

**Comment by serengil** [comment]
Must be a bug in your flask's version. Because we have units tests for that feature. Try to use the same version from github actions.

**Comment by serengil** [comment]
Additionally, when i tested the following command in my environment, I am getting a result

```shell
curl -X POST http://localhost:5005/verify -H "Content-Type:
multipart/form-data" -F "model_name=Facenet" -F "img1=@img1.jpg" -F "img2=@img2.jpg"
```

Result:

```json
{"detector_backend":"opencv","distance":0.236863,"facial_areas":{"img1":{"h":768,"left_eye":[850,524],"right_eye":[571,517],"w":768,"x":339,"y":218},"img2":{"h":491,"left_eye":[858,388],"right_eye":[663,390],"w":491,"x":524,"y":201}},"model":"Facenet","similarity_metric":"cosine","threshold":0.4,"time":1.14,"verified":true}
```

These are the dependencies, please confirm that you are using the same versions.

```
deepface==0.0.94
Flask==2.0.2
Werkzeug==2.0.2
```

**Comment by serengil** [comment]
I can reproduce the problem with Flask 3.

Please pull the latest code and re-run. 

Related change: https://github.com/serengil/deepface/commit/4fe4ba289e9ee78c61337af0238b7236ad0c17ee
